src/Components/Attributes/test/Attributes.js

```diff
@@ -57,14 +54,14 @@ describe('Attributes', () => {
     }, { id: 'Address' });
     const userObject = new minim.ObjectElement({
       name: 'Doe',
-      address: new ReferenceElement(addressObject.id),
+      address: createReferenceElement(addressObject.id),
     }, { id: 'User' });
 
     const dataStructures = new Category(
-      new minim.ArrayElement([
```

Comment:
@kylef The test get fixed by just passing a plain array of DataStructures. But this looks like an undocumented backward's incompatible change in minim.

Reply:
Ah, I just read the https://github.com/apiaryio/attributes-kit/pull/478#pullrequestreview-190274148. So this is correct now.